tests/back/api/test_dataloaders.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+from DashAI.back.registries import DataloaderRegistry, TaskRegistry
+from DashAI.back.tasks import BaseTask
+from DashAI.back.dataloaders import BaseDataLoader
+from fastapi.testclient import TestClient
+from DashAI.back.core import config
+
+default_dataloader_dict = {'foo': 'bar'}
```

Comment:
What is the purpose of this dict?

Reply:
This dict was meant to be a test for a dict like this:
```
{
    "class": "JSONDataLoader",
    "name": "JSON Data",
    "help": "Use JSON files to upload the data. You can use a .json file or multiple .json files in a .zip file.",
    "description": "You can upload your data in a .json file or in multiple .json files in a .zip, where you can have the definition of splits in folders as shown above. \n If you only have one file or multiple files without the folder definition for the splits, you can set them up later. \n The JSON files have to follow the format that specify in the last figure, the data have to be in a key, that contains an array with the samples of the data or rows in JSON objects, where each one have the features of data or columns like key/value pairs. \n Make sure that all the JSON files have the same features and format.",
    "images": ["/info_images/json_files.png", "/info_images/json_format.svg"],
	"type": "object"
}
```
I replaced it with a more explicit `{'class':'DataLoader1'}` inside the DataLoader, get_schema